Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableOnTouchReceiver.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// A basic touch event receiver for detecting Physical Touch state changes in the Interactable
+    /// When the physical touch states change, these events are triggered.
+    /// </summary>
+    public class InteractableOnTouchReceiver : ReceiverBase
```

Comment:
I'm not able to create a receiver that responds to touch events. Could you provide and example of this in the scene? I was able to get grab receiver working.

Reply:
I added an InteractableOnTouchReciever to the NearInteractionSample, the cube with the white faceted sphere inside.

As for the voice command stuff, I was totally confused. There used to be voice recognition code in Interactable that was removed, but I didn't know that. The voice command field would enable a voice command directly from Interactable, but now that it's gone the feilds are confusing. Like the Requires Gaze doesn't really do anything now, so it needs to be removed. The Voice Command field does filter the voice commands, but it should be a drop down pulling a list from the speech input profile. I'll open an issue for this.

Reply:
The documentation for Voice Commands in Interactable is totally wrong now and needs to point to the Speech Input setup.